.github/workflows/xk6.yml

```diff
@@ -0,0 +1,57 @@
+name: xk6
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-xk6:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
+        platform: [ubuntu-latest, windows-latest]
```

Comment:
not certain it matters all that much to test with the previous version as well, but maybe test it with macos ?

Reply:
hmm given https://github.com/k6io/xk6/issues/9, we should probably test it with the newer Go versions instead, but let's switch after we fix that 1.16 issue and release k6 v0.31.0

Reply:
but yeah, :+1: for macos, I'll add it

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1885/commits/6dfbcc30d09fc53d9428a6f81e81a2b61a62466e